brambling/management/commands/send_daily_emails.py

```diff
@@ -0,0 +1,41 @@
+import datetime
+import traceback
+
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from brambling.mail import DailyDigestMailer
+from brambling.models import Person
+
+
+class Command(BaseCommand):
+    def get_recipients(self):
+        return Person.objects.filter(
+            notify_new_purchases=Person.NOTIFY_DAILY,
```

Comment:
That filtering happens as part of the Mailer. It only gets transactions that happened since last_new_purchases_digest_sent (up to a max of 1 day). This is where the cutoff in this command's send_digest() method comes in.